citations/citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -1336,28 +1336,28 @@ protected Map<String,Object> ensureCitationListExists(ParameterParser params,
 
 						getContentService().commitResource(edit, priority);
 						resourceId = edit.getId();
-						message =  rb.getFormattedMessage("resource.new.success", new String[]{ displayName });
+						message =  rb.getFormattedMessage("resource.new.success", (Object) new String[]{ displayName });
 					} catch (IdUniquenessException e) {
 						message = e.getMessage();
-						log.warn("IdUniquenessException in ensureCitationListExists() {}", e);
+						log.warn("IdUniquenessException in ensureCitationListExists()", e);
 					} catch (IdLengthException e) {
 						message = e.getMessage();
-						log.warn("IdLengthException in ensureCitationListExists() {}", e);
+						log.warn("IdLengthException in ensureCitationListExists()", e);
 					} catch (IdInvalidException e) {
 						message = e.getMessage();
-						log.warn("IdInvalidException in ensureCitationListExists() {}", e);
+						log.warn("IdInvalidException in ensureCitationListExists()", e);
 					} catch (OverQuotaException e) {
 						message = e.getMessage();
-						log.warn("OverQuotaException in ensureCitationListExists() {}", e);
+						log.warn("OverQuotaException in ensureCitationListExists()", e);
 					} catch (ServerOverloadException e) {
 						message = e.getMessage();
-						log.warn("ServerOverloadException in ensureCitationListExists() {}", e);
+						log.warn("ServerOverloadException in ensureCitationListExists()", e);
 					} catch (PermissionException e) {
 						message = e.getMessage();
-						log.warn("PermissionException in ensureCitationListExists() {}", e);
+						log.warn("PermissionException in ensureCitationListExists()", e);
 					} catch (IdUnusedException e) {
 						message = e.getMessage();
-						log.warn("IdUnusedException in ensureCitationListExists() {}", e);
+						log.warn("IdUnusedException in ensureCitationListExists()", e);
```

Comment:
Simply catching Exception and then using the exceptions toString will include the exception class name in the message, yielding the same result in the log message.

Reply:
I would also advocate for a better message, as the current message(s) are unhelpful and expected.

Reply:
idk this isn't the scope of the change. i'm trying to be helpful and cleanup stuff without refactoring logging in a class that is never used ;)